npm_test.go

```diff
@@ -437,6 +439,9 @@ func TestYarn(t *testing.T) {
 	defer cleanUpYarnGlobalFolder()
 
 	jfrogCli := tests.NewJfrogCli(execMain, "jfrog", "")
+	yarnExecPath, err := exec.LookPath("yarn")
+	assert.NoError(t, err)
+	yarn.ConfigSet("unsafeHttpWhitelist", "[\"localhost\"]", yarnExecPath, true)
```

Comment:
1. Handle error.
2. We might want to defer set the original value back. You can use ConfigGet to get the original.
3. Not sure square brackets are needed.